test/install_agent_6_macos.yaml

```diff
@@ -0,0 +1,27 @@
+---
+
+- hosts: all
+  tasks:
+    - ansible.builtin.import_role:
+        name: datadog.dd.agent
+  vars:
+    datadog_api_key: "11111111111111111111111111111111"
+    datadog_enabled: false
+    datadog_agent_major_version: 6
+    # avoid checking that the agent is stopped for centos
+    datadog_skip_running_check: true
+    datadog_config:
+      tags: "mytag0, mytag1"
+      log_level: INFO
+      apm_enabled: "true"  # has to be set as a string
+    datadog_config_ex:
+      trace.config:
+        env: dev
+      trace.concentrator:
+        extra_aggregators: version
+    datadog_checks:
```

Comment:
should we enable different checks somewhere to "increase coverage"?

Reply:
We could, but I think it should be done in a follow up card after deciding what scenari we want to test